static/js/components/ExpandedPostDisplay.js

```diff
@@ -119,7 +134,8 @@ export default class ExpandedPostDisplay extends React.Component<Props> {
         </div>
         <div className="right">
           {SETTINGS.username === post.author_id &&
-          (post.text || post.article_content) ? (
+          (post.post_type === LINK_TYPE_TEXT ||
+            post.post_type === LINK_TYPE_ARTICLE) ? (
```

Comment:
Also take or leave: I'm in favor of the other cases where we explicitly check the `post_type` value, but here I think it would be a little more readable and reusable to create a helper function. `isEditablePostType` or something like that

Reply:
sure, then we don't have to edit every place we might want to have some sort of editing interface later.